src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -177,6 +177,22 @@ internal static bool IsInbox
 
 #endif
 
+        /// <summary>
+        /// True if it is an instance of PowerShell Core
+        /// </summary>
+        public static bool IsPowerShellCore
```

Comment:
Can we use [IsCoreCLR](https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/CoreCLR/CorePsPlatform.cs#L69)?